src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/EvidenceReceivedFunctionalTest.java

```diff
@@ -0,0 +1,123 @@
+    private String oralEvidenceReceivedRepsSmsId;
+
+    public EvidenceReceivedFunctionalTest() {
+        super(30);
+    }
+
+    @Test
+    @Parameters({"ORAL", "PAPER"})
+    public void givenEvidenceReceivedWithRepsSubscription_shouldSendNotificationToReps(AppealHearingType appealHearingType)
+            throws Exception {
+        //Given
+        final String appellantEmailId = getFieldValue(appealHearingType, "AppellantEmailId");
+        final String appellantSmsId = getFieldValue(appealHearingType, "AppellantSmsId");
+        final String repsEmailId = getFieldValue(appealHearingType, "RepsEmailId");
+        final String repsSmsId = getFieldValue(appealHearingType, "RepsSmsId");
+
+        simulateCcdCallback(EVIDENCE_RECEIVED_NOTIFICATION,
+                "representative/"
+                        + (appealHearingType != null ? (appealHearingType.name().toLowerCase() + "-") : "")
+                        + EVIDENCE_RECEIVED_NOTIFICATION.getId() + "Callback.json");
+
+        final List<Notification> notifications = (StringUtils.isBlank(appellantSmsId))
+                ? tryFetchNotificationsForTestCase(appellantEmailId, repsEmailId, repsSmsId)
+                : tryFetchNotificationsForTestCase(appellantEmailId, appellantSmsId, repsEmailId, repsSmsId);
+
+        assertNotificationBodyContains(notifications, appellantEmailId);
+        if (!PAPER.equals(appealHearingType)) {
+            assertNotificationBodyContains(notifications, appellantSmsId);
+        }
+
+        String representativeName = "Harry Potter";
+        assertNotificationBodyContains(notifications, repsEmailId, representativeName);
+        assertNotificationBodyContains(notifications, repsSmsId);
+    }
+
+    @Test
+    @Parameters({"ORAL", "PAPER"})
+    public void givenEvidenceReceivedWithNoRepSubscription_shouldNotSendNotificationToReps(AppealHearingType appealHearingType)
+            throws Exception {
+
+        final String appellantEmailId = getFieldValue(appealHearingType, "AppellantEmailId");
+        final String appellantSmsId = getFieldValue(appealHearingType, "AppellantSmsId");
+        final String repsEmailId = getFieldValue(appealHearingType, "RepsEmailId");
+        final String repsSmsId = getFieldValue(appealHearingType, "RepsSmsId");
+
+        simulateCcdCallback(EVIDENCE_RECEIVED_NOTIFICATION,
+                "representative/" + "no-reps-subscribed-"
+                        + appealHearingType.name().toLowerCase() + "-"
+                        + EVIDENCE_RECEIVED_NOTIFICATION.getId() + "Callback.json");
+
+        List<Notification> notifications = tryFetchNotificationsForTestCase(appellantEmailId,
+                    appellantSmsId);
+        assertNotificationBodyContains(notifications, appellantEmailId);
+        assertNotificationBodyContains(notifications, appellantSmsId);
+        List<Notification> notificationsNotFound = tryFetchNotificationsForTestCaseWithFlag(true,
```

Comment:
This variable name is misleading.

Reply:
it makes sense to me. how is it misleading?

Reply:
Is this not a list of found notifications?

Reply:
No. They shouldn't be found. (btw: I didn't write this).